lib/function_loader.js

```diff
@@ -32,6 +32,7 @@ function checkConflicts(obj1, obj2) {
 module.exports = function(eyeglass, sass, options, existingFunctions) {
   var root = options.root;
   var functions = {};
+  hash.merge(functions, require("./functions")(eyeglass, sass));
```

Comment:
That seems reasonable. Dangerous, but reasonable. We probably need some way to get at the old function for passthrough purposes. Doesn't block this PR